packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/TypeFactory.cs

```diff
@@ -220,14 +217,7 @@ internal CSharpType CreatePrimitiveCSharpType(InputType inputType)
         /// <param name="parameter">The <see cref="InputParameter"/> to convert.</param>
         /// <returns>An instance of <see cref="ParameterProvider"/>.</returns>
         public virtual ParameterProvider CreateParameter(InputParameter parameter)
```

Comment:
Should we have the protected CreateParameterCore method and make the public method not be virtual?

Reply:
Nevermind - on the protected method, but this should probably not be virtual.